src/client/tile.cpp

```diff
@@ -537,6 +537,34 @@ static void apply_colorize(video::IImage *dst, v2u32 dst_pos, v2u32 size,
 static void apply_multiplication(video::IImage *dst, v2u32 dst_pos, v2u32 size,
 		const video::SColor &color);
 
+// Perform a Screen blend with the given color. The opposite effect of a
+// Multiply blend.
+static void apply_screen(video::IImage *dst, v2u32 dst_pos, v2u32 size,
+		const video::SColor &color);
+
+// Adjust the hue, saturation, and lightness of destination. Like
+// "Hue-Saturation" in GIMP, but with 0 being the mid-point.
+// If colorize is true then all pixels will be converted to the specified hue,
+// but retain their saturation and lightness, like "Colorize" in GIMP.
+// Hue should be from -180 to +180, or from 0 to 360.
+// Saturation and lightness are both from -100 to +100
+static void apply_hue_saturation(video::IImage *dst, v2u32 dst_pos, v2u32 size,
+		int hue, int saturation, int lightness, bool colorize);
+
+// Apply an overlay blend to an images.
+// Overlay blend combines Multiply and Screen blend modes.The parts of the top
+// layer where the base layer is light become lighter, the parts where the base
+// layer is dark become darker.Areas where the base layer are mid grey are
+// unaffected.An overlay with the same picture looks like an S - curve.
+static void apply_overlay(video::IImage *overlay, video::IImage *dst,
+		v2s32 overlay_pos, v2s32 dst_pos, v2u32 size, bool hardlight);
+
+// Adjust the brightness and contrast of the base image. Conceptually like
+// "Brightness-Contrast" in GIMP but allowing brightness to be wound all the
+// way up to white or down to black.
```

Comment:
I think you could leave the "like XXX in GIMP" comparisons out of the function docs here